test/helpers.bash

```diff
@@ -250,6 +250,12 @@ function setup_crio() {
     CNI_DEFAULT_NETWORK=${CNI_DEFAULT_NETWORK:-crio}
     CNI_TYPE=${CNI_TYPE:-bridge}
 
+    # Workaround for https://github.com/containers/crun/pull/531.
+    # TODO: remove once crun 0.16 is released and used here.
```

Comment:
@kolyshkin, based on https://github.com/cri-o/cri-o/pull/4293#issuecomment-720217456, wouldn't be better to update the comment here?

Let me know what's your preference and I'll label the PR accordingly. :-)

Reply:
fixed (> 0.15 is the correct way to say it)